frontend/components/commons/results/ResultsList.vue

```diff
@@ -16,63 +16,69 @@
+                <slot name="record" :record="item" />
+              </results-record>
+            </DynamicScrollerItem>
+          </template>
+          <template #after>
+            <pagination-end-alert :limit="paginationLimit" v-if="isLastPagePaginable" />
+          </template>
+        </DynamicScroller>
+        <LazyReModal
+          modal-class="modal-secondary"
+          modal-position="modal-center"
+          :modal-custom="true"
+          :prevent-body-scroll="true"
+          :modal-visible="selectedRecord !== undefined"
+          @close-modal="onCloseMetadata"
+        >
+          <Metadata
+            v-if="selectedRecord"
+            :applied-filters="dataset.query.metadata"
+            :metadata-items="selectedRecord.metadata"
+            :title="selectedRecord.recordTitle()"
+            @metafilterApply="onApplyMetadataFilter"
+            @cancel="onCloseMetadata"
+          />
+        </LazyReModal>
       </div>
-      <DynamicScroller
-        v-else
-        page-mode
-        class="scroller"
-        :items="visibleRecords"
-        :min-item-size="150"
-        :buffer="200"
-        :style="{ paddingTop: `${dataset.viewSettings.headerHeight + 10}px` }"
-      > 
-        <template v-slot="{ item, index, active }">
-          <DynamicScrollerItem
-            :watch-data="true"
-            class="list__li"
-            :item="item"
-            :active="active"
-            key-field="id"
-            :index="index"
-            :data-index="index"
-          >
-            <results-record @show-metadata="onShowMetadata" :key="item.id" :dataset="dataset" :item="item">
-              <slot name="record" :record="item" />
-            </results-record>
-          </DynamicScrollerItem>
-        </template>
-        <template #after>
-          <pagination-end-alert :limit="paginationLimit" v-if="isLastPagePaginable" />
-        </template>
-      </DynamicScroller>
-      <LazyReModal
-        modal-class="modal-secondary"
-        modal-position="modal-center"
-        :modal-custom="true"
-        :prevent-body-scroll="true"
-        :modal-visible="selectedRecord !== undefined"
-        @close-modal="onCloseMetadata"
-      >
-        <Metadata
-          v-if="selectedRecord"
-          :applied-filters="dataset.query.metadata"
-          :metadata-items="selectedRecord.metadata"
-          :title="selectedRecord.recordTitle()"
-          @metafilterApply="onApplyMetadataFilter"
-          @cancel="onCloseMetadata"
-        />
-      </LazyReModal>
-    </div>
-    <RePagination
-      :total-items="dataset.results.total"
-      :pagination-settings="dataset.viewSettings.pagination"
-      @changePage="onPagination"
-    />
+      <RePagination
+        :total-items="dataset.results.total"
+        :pagination-settings="dataset.viewSettings.pagination"
+        @changePage="onPagination"
+      />
+    </template>
+    <rules-summary v-else :dataset="dataset"/>
```

Comment:
common components must contain only common behaviour. Please, refactor and reuse common components in specific views